core/sqf/sql/scripts/dcscheck

```diff
@@ -73,14 +72,14 @@ if ( [ $sq_stat == 0 ] || [ $sq_stat == 1 ] ); then
 
 ### Get the configured primary node for DcsMaster
   if [ -s ${DCS_INSTALL_DIR}/conf/master ]; then
```

Comment:
Should this statement be changed to conf/masters? If not, should this "if" be deleted?

Reply:
Yes this should be deleted. will take care of it . Thanks